### ISSUE TITLE
get the result of Rank-1 accuracy

### ISSUE THREAD

**Issue by LinlyAC** [deepinsight/insightface#473]
Hi, @nttstar . Very nice work!  I am a newbie for face recognition.  I've fine-tuned my own dataset with pre-trained models. Now I want to get the rank1 accuracy of my model on the validation set, but I only found accuracy and VR@FAR in src/eval/verification.py. So how to get the result of  rank-1 by your code?
I look forward to your reply, thank you very much.
